Apps/RaptureIntTests/src/test/resources/local_testng.xml

```diff
@@ -13,6 +13,9 @@
 			<package name="rapture.httpapi.series" />
 			<package name="rapture.httpapi.document" />
 			<package name="rapture.httpapi.plugin" />
+			<package name="rapture.nightly.java" />
```

Comment:
Currently it's where the java entitlement test lives. I will redo the packages if for no other reason than the java part is redundant... the tests are under src/java.